config/env_config.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"log"
+	"os"
+	"web/model"
+
+	"github.com/joho/godotenv"
+)
+
+func ConfigureEnvironmentals() (model.AppConfig, model.PostgresConfig) {
```

Comment:
Still named as configureenvironmentals. Did you decide to still leave it as it is?

Reply:
I think the proper name should be LoadEnvironmentals. NewEnvironmentals as mentioned in slack can cause some confuse, because we're not making newEnvironment Variables, we're loading the values of existing ones.